ubiconfig/utils/api/gitlab.py

```diff
@@ -38,13 +38,13 @@ def api_url(self):
     def get_branch_list_api(self):
         return urljoin(self.api_url, 'repository/branches')
 
-    def get_file_list_api(self, branch=None, per_page=30):
+    def get_file_list_api(self, branch=None, page=1):
```

Comment:
Minor: I reckon you probably shouldn't even put a default value for page here.

It's minor since this isn't public API.  But really you want to force the caller to think about pages.  If you give a default, it's too easy for someone to just write get_file_list_api() and not even think about pagination until it's too late.  Omitting the default forces them to think about pagination up-front as they should.

Reply:
yeah, I overlooked this. 
I'll remove this default value.